tools/android_custom_build/build_custom_android_package.py

```diff
@@ -136,7 +145,7 @@ def main():
         args.docker_path,
         "run",
         "--rm",
-        "-it",
+        f"-{args.docker_interactive_mode_flag}",
```

Comment:
this offers a bit too much flexibility - a user can pass an arbitrary value and perhaps specify other docker options.
if you want to make this configurable with an option, would suggest just having that option control whether "-it" or "-i" is used.

Reply:
I agree. If the choice is between `-it` and `-i` it's about whether the psuedo TTY is created isn't it, so maybe an arg called docker_psuedo_tty with default of True to match current behaviour?

Reply:
This proposal sounds good, I will make this argument as a bool so we only change between the "-it" and "-i" option.

Reply:
I've addressed this in the commit [92b0197](https://github.com/microsoft/onnxruntime/pull/13704/commits/92b019772a0c1b90c614eb3124df8a8290db7cad), meanwhile, as we have decided to go with removing -it flag directly, this would not be part of the change, thus resolving this.